lib/ABI/FunctionType/Conversion.cpp

```diff
@@ -405,13 +405,16 @@ TCC::tryConvertingStackArguments(model::TypePath StackArgumentTypes,
     return llvm::SmallVector<model::Argument, 8>{};
   }
 
-  model::StructType &Stack = *llvm::cast<model::StructType>(StackArgumentTypes
-                                                              .get());
+  auto &Stack = *llvm::cast<model::StructType>(StackArgumentTypes.get());
 
-  // Compute the full alignment.
-  model::QualifiedType StackStruct = { StackArgumentTypes, {} };
-  uint64_t StackAlignment = *ABI.alignment(StackStruct);
+  // As a workaround for the cases where expected alignment is missing at
+  // the very end of the RFT-style stack argument struct, use adjusted
+  // size and alignment values instead of the real ones.
+  uint64_t StackAlignment = *ABI.alignment(Stack);
```

Comment:
@aleclearmind, please review the new `ToCFT: fix last stack argument verification` commit, it doesn't have anything to do with this PR, but it feels like this is the best place for it to get a piggy-back ride.